jaeger-core/src/test/java/com/uber/jaeger/reporters/RemoteReporterTest.java

```diff
@@ -68,10 +72,13 @@ public void setUp() throws Exception {
   public void testRemoteReporterReport() throws Exception {
     Span span = (Span) tracer.buildSpan("raza").start();
     reporter.report(span);
-    Thread.sleep(50);
+    // do sleep until automatic flush happens on 'reporter'
+    // added 20ms on top of 'flushInterval' to avoid corner cases
+    await().with().pollInterval(50, TimeUnit.MILLISECONDS).atMost(
+            flushInterval + 20, TimeUnit.MILLISECONDS).until(() -> sender.getReceived().size() > 0);
```

Comment:
the flush interval at L53 is 1000ms, which is too long for the tests. But here the previous test was only sleeping for 50ms, so it's not clear to me how it was working to begin with.

Per comment below, the poll interval can be 1ms. Basically, the goal is to speed up the test as much as possible, while keeping it reliable.